src/python/pants/option/global_options.py

```diff
@@ -145,7 +145,7 @@ def register_bootstrap_options(cls, register):
 
     # These logging options are registered in the bootstrap phase so that plugins can log during
     # registration and not so that their values can be interpolated in configs.
-    register('-d', '--logdir', advanced=True, metavar='<dir>',
+    register('-d', '--logdir', advanced=True, metavar='<dir>', daemon=False,
```

Comment:
I've never seen this one configured on the CLI, so it feels like it would be fine to continue to include it in the fingerprint.

Reply:
k, it didn't seem relevant to the daemon to me since it's only used for logging run output - but, restored.